### PR TITLE
Added more tests

### PR DESCRIPTION
**Pull request by gulyapulya**
Added multiple tests cases for new and currently tested functions.

### CODE REVIEW

File: src/tests/posts.test.js [hunk 1]
```diff
@@ -16,9 +16,25 @@ describe("posts tests", () => {
...
+  test("if null is passed, expect true", () => {
+    expect(isNullOrWhitespace(null)).toBeTruthy();
+  });
+
+  test("if whitespace is passed, expect true", () => {
+    expect(isNullOrWhitespace(" ")).toBeTruthy();
+  });
 });
 
 describe("snapshot testing", () => {
```
Comment (mnosov622) [inline src/tests/posts.test.js:40]: Hi !

Is it supposed to be here ? I don't see any use for it. Could you remove it, please ?

**Review by mnosov622 — CHANGES_REQUESTED** [review]
Is it supposed to be here ? I don't see any use for it. Could you remove it, please ?

File: src/tests/posts.test.js (hunk 1 above)
Comment (mnosov622) [inline src/tests/posts.test.js:40]: Is it supposed to be here ? I don't see any use for it. Could you remove it, please ?

**Review by mnosov622 — CHANGES_REQUESTED** [review]

File: src/tests/posts.test.js (hunk 1 above)
Comment (mnosov622) [reply src/tests/posts.test.js:40]: Line 40

Comment (gulyapulya) [reply src/tests/posts.test.js:40]: There are more lines than 40 in the file, they were just collapsed bcs I did not change anything there, you can open them by clicking blue arrow at the end of the file.

**Comment by mnosov622** [comment]
Looks great ! Thank you :)

**Pull request merged**